db/migrations/20210111183822_get_block_heights_for_transformed_diffs.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+
+CREATE OR REPLACE FUNCTION api.get_block_heights_for_transformed_diffs()
+    RETURNS SETOF BIGINT
+    LANGUAGE sql
+    STABLE
+AS
+$$
+SELECT block_height FROM public.storage_diff WHERE status = 'transformed' ORDER BY block_height ASC
```

Comment:
Is it useful to get back a whole set here, or should we just limit to the latest? I don't feel super strongly either way—if the past few block heights are useful in some way, keep them.

Reply:
Yeah I think we probably only need the maximum block height of any transformed diff. I believe that this query would return millions of rows

Reply:
Ah makes sense. I think I misunderstood how the query would be used with `last` argument to filter down the collection.

Reply:
Yeah, that's a common gotcha with postgraphile - the `first`/`last` params aren't very helpful if being used on a function call: https://www.graphile.org/postgraphile/custom-queries/#advice